src/lang/res_ja-JP.rc

```diff
@@ -410,6 +417,7 @@ JAPANEND
 
     IDS_DRAG_COPYING            "%s へファイルをコピー中"
     IDS_DRAG_MOVING             "%s へファイルを移動中"
+    IDS_DRAG_LINKING            "ファイルを ％s にリンクする"

```

Comment:
As mentioned by PR #355 , some full width `％` symbols were introduced here by mistake and should be corrected then...